src/DurableTask.Netherite/DurableTask.Netherite.csproj

```
@@ -56,7 +56,7 @@
     <PackageReference Include="Microsoft.Azure.EventHubs.Processor" Version="4.3.2" />
     <PackageReference Include="Microsoft.Azure.Storage.Blob" Version="11.2.3" />
     <PackageReference Include="Azure.Storage.Blobs" Version="12.16.0" />
-	<PackageReference Include="Microsoft.FASTER.Core" Version="2.0.23" />
+	<PackageReference Include="Microsoft.FASTER.Core" Version="2.6.1" />
```

```suggestion
	<PackageReference Include="Microsoft.FASTER.Core" Version="2.6.2" />
```

FASTER v2.6.2 has been released.